cluster/manifests/skipper/deployment.yaml

```diff
@@ -323,21 +323,18 @@ spec:
           - "-forwarded-headers=X-Forwarded-For,X-Forwarded-Proto=https,X-Forwarded-Port=443"
           - '-forwarded-headers-exclude-cidrs={{ .cluster_internal_cidrs | join "," }}'
 {{ end }}
-          - >-
```

Comment:
Config is multiline so I propose we keep newlines to make actual value close to the configured.